src/components/TaskBoard/TaskInput.js

```diff
@@ -62,6 +62,14 @@ const TaskTimeInputDiv = styled.div`
         font-size: 1.2em;
         color: #fabb18;
     }
+    input[type="number"] { 
```

Comment:
if you notice, i have already put the code to make the arrows disappear from webkit based browsers. the code is in TaskTimeInputField component (the one below which you have changed). i think putting the fix for firefox should there should work. As of now, in the deploy preview, i am seeing the arrow buttons

Reply:
Removed code that make arrows disappear from webkit based browsers (since you already added the code) and moved the firefox fix to TaskTimeInputDiv. This should ensure the fix is working in the deploy preview.